### PR TITLE
kava 5 types

### PR DESCRIPTION
**Pull request by lukanus**

### CODE REVIEW

File: api/mapper/hard.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+	se = shared.SubsetEvent{
+		Type:   []string{"hard_repay"},
+		Module: "hard",
+		Node: map[string][]shared.Account{
+			"keeper":   {{ID: bech32Addr}},
+			"borrower": {{ID: bech32BorrowerAddr}},
+		},
+	}
+
+	err = produceTransfers(&se, "send", logf)
```
Comment (lukanus) [inline api/mapper/hard.go:144]: @pacmessica Not sure about those "send" types, if this correct, you should be probably able to tell easier than me

File: README.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ Worker also need some basic config:
 
 ```bash
     MANAGERS=0.0.0.0:8085
-    TENDERMINT_RPC_ADDR=https://kava-3--rpc--archive.datahub.figment.io
-    TENDERMINT_LCD_ADDR=https://kava-3--rpc--archive.datahub.figment.io
-    CHAIN_ID=kava-3
```
Comment (pacmessica) [inline README.md:34]: did we jump from 3 to 5? what happened to 4 😱

File: api/transaction.go [hunk 3]
```diff
@@ -310,23 +312,29 @@ func rawToTransaction(ctx context.Context, c *Client, in []types.TxResponse, blo
...
+					ev, err = mapper.HardWithdrawToSub(msg, logAtIndex)
+				case "hard_borrow":
+					ev, err = mapper.HardBorrowToSub(msg, logAtIndex)
+				case "hard_liquidate":
+					ev, err = mapper.HardLiquidateToSub(msg, logAtIndex)
+				case "hard_repay":
+					ev, err = mapper.HardRepayToSub(msg, logAtIndex)
 				default:
-					c.logger.Error("[COSMOS-API] Unknown harvest message Type ", zap.Error(err), zap.String("type", msg.Type()), zap.String("route", msg.Route()))
+					c.logger.Error("[COSMOS-API] Unknown hard message Type ", zap.Error(err), zap.String("type", msg.Type()), zap.String("route", msg.Route()))
```
Comment (pacmessica) [inline api/transaction.go:328]: let's add `height` to logs so it's easier to debug

**Review by pacmessica — COMMENTED** [review]
looks good :)

**Comment by lukanus** [comment]
Invalid for now

**Pull request closed without merging**